### PR TITLE
documentation(EJ2-847685):Added docs for load on demand and single page exporting

### PR DESCRIPTION
**Pull request by lokesharjunan**

### CODE REVIEW

File: ej2-javascript/gantt/data-binding.md [hunk 1]
```diff
@@ -171,6 +171,376 @@ public ActionResult UrlDatasource(DataManagerRequest dm)
 
 ```
 
+### Load Child on Demand
```
Comment (MahalakshmiKarthikeyan) [inline ej2-javascript/gantt/data-binding.md:174]: in title first letter alone should be in caps

File: ej2-javascript/gantt/pdf-export/pdf-export.md [hunk 2]
```diff
@@ -81,4 +81,37 @@ step 2: Then , `pdfExpComplete` return as blob object.
 {% endtabs %}
 
 {% previewsample "page.domainurl/code-snippet/gantt/exporting-blob-data-cs1" %}
+{% endif %}
+
+## Single Page Exporting in Gantt
```
Comment (MahalakshmiKarthikeyan) [inline ej2-javascript/gantt/pdf-export/pdf-export.md:86]: in title first letter alone should be in caps

File: ej2-javascript/gantt/data-binding.md (hunk 1 above)
Comment (lokesharjunan) [reply ej2-javascript/gantt/data-binding.md]: Resolved

File: ej2-javascript/gantt/pdf-export/pdf-export.md (hunk 2 above)
Comment (lokesharjunan) [reply ej2-javascript/gantt/pdf-export/pdf-export.md]: Resolved

**Review by MahalakshmiKarthikeyan — APPROVED** [review]

**Pull request merged**